src/MSBuild/Resources/xlf/Strings.it.xlf

```diff
@@ -162,7 +162,7 @@ Questo flag è sperimentale e potrebbe non funzionare come previsto.
       </trans-unit>
       <trans-unit id="MSBuildVersionMessage">
         <source>MSBuild version {0} for {1}</source>
-        <target state="translated">Versione di MSBuild {0} per {1}</target>
+        <target state="translated">Versione di MSBuild Ł{0} per {1}</target>
```

Comment:
I distinctly remember fixing this this recently. I'm going to manually override this because the regression bugs me.